DomainContent/FaceToFace/StageManager/enterStageZone.js

```diff
@@ -0,0 +1,115 @@
+//
+//  enterStageZone.js
+//
+//  Created by Rebecca Stankus on 06/16/18
+//  Copyright 2018 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+(function() {
+
+    var overlays = [];
+    var arrowLeft;
+    var arrowRight;
+    this.enterEntity = function() {
+        var webOverlay = Overlays.addOverlay("web3d", {
+            dimensions: {
+                x: 1.3249601125717163,
+                y: 0.7452900409698486,
+                z: 0.009999999776482582
+            },
+            rotation: {
+                w: 0.0014801025390625,
+                x: -0.0007781982421875,
+                y: 0.9099717140197754,
+                z: 0.4146333932876587
+            },
+            position: {
+                x: 99.6183,
+                y: -0.6548,
+                z: 32.4620
+            },
+            url: "https://hifi-slides.glitch.me/",
```

Comment:
This is fine for now, but we might end up hitting some weirdness if we end up putting an entity whitelist up and miss that this is on glitch instead of our S3 bucket.